t/t-attributes.sh

```diff
@@ -55,3 +55,26 @@ begin_test "macros with HOME"
   assert_local_object "$(calc_oid "$contents")" 9
 )
 end_test
+
+begin_test "macros with HOME split"
+(
+  set -e
+
+  reponame="$(basename "$0" ".sh")-home-split"
+  clone_repo "$reponame" repo-home-split
+
+  mkdir -p "$HOME/.config/git"
+  printf '[attr]lfs filter=lfs diff=lfs merge=lfs -text\n' \
+    > "$HOME/.config/git/attributes"
+
+  printf '*.dat lfs\n' > .gitattributes
+  git add .gitattributes
+  git commit -m 'initial import'
+
+  contents="some data"
+  printf "$contents" > foo.dat
+  git add *.dat
+  git commit -m 'foo.dat'
+  assert_local_object "$(calc_oid "$contents")" 9
```

Comment:
Same here as above: I think we need a `git lfs track` call to ensure the pattern is already supported